### DISCUSSION TITLE
Energy Measurement (E): improve clarity and consistency

### DISCUSSION THREAD

**Discussion by dschien** [Green-Software-Foundation/sci#192] [SCI feedback]
1. review use of term "measurement"
2. Clarify in what sense is the list provided a list of "taxonomies"? What sort of "taxonomy" is a data centre? 
3. Check consistency in examples of provided for "infrastructure and systems" - how are "Individual service (e.g. API call, ML training job)" and "Execution of code" example of infrastructure and systems?

**Comment by atg-abhishek** [comment]
Can you create a PR to address these once the existing PRs are merged in during today's WG call so that you have the latest? Thanks!
